openapi/reputation/reputation.yaml

```diff
@@ -0,0 +1,195 @@
+          required: true
+        - schema:
+            type: string
+          in: query
+          name: 'filter[businessLocation.id]'
+          description: Return reviews for the specified business location
+          required: true
+        - schema:
+            type: integer
+          in: query
+          name: 'filter[businessListing.id]'
+          description: Return reviews for a specific business listing
+        - schema:
+            type: integer
+          in: query
+          name: 'filter[listingSource.id]'
+          description: Return reviews for a listing source
+        - schema:
+            type: string
+            format: date-time
+          in: query
+          name: 'filter[startAt]'
+          description: Return reviews published on or after this date
+        - schema:
+            type: string
+            format: date-time
+          in: query
+          name: 'filter[endAt]'
+          description: Return reviews published on or before this date
+        - schema:
+            type: string
+          in: query
+          name: 'filter[rating]'
+          description: Return reviews for the listing source
+        - schema:
+            type: string
+          in: query
+          name: 'filter[phrase]'
+          description: A phrase to find reviews for
+        - schema:
+            type: integer
+            minimum: 0
+            maximum: 1000
+            default: 10
+          in: query
+          name: 'page[limit]'
+          description: The maximum number of tasks you would like returned in a single batch. Use the links.next member in the response to get the remainder.
+        - schema:
+            type: string
+          in: query
+          name: 'page[cursor]'
+          description: The cursor stores all your filters and current location in the list to allow paging over the results in smaller batches. The value will be provided in the response links.
+        - schema:
+            type: string
+          in: header
+          name: Accept-Language
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/businessLocationReviews'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+                        format: uri
+                      first:
+                        type: string
+                        description: Provides a link back to the first page of results
+                        format: uri
+                      next:
+                        type: string
+                        description: The URI at which the next batch of purchases can be gotten from
```

Comment:
purchases -> reviews

I want to take a moment to say that I definitely am not (am) copy and pasting as well....maybe I should c+p the purchases endpoint